janusgraph-dist/src/assembly/static/cassandra/conf/cassandra.yaml

```diff
@@ -31,26 +37,23 @@ cluster_name: 'Cassandra testing instance'
 # If you leave this unspecified, Cassandra will use the default of 1 token for legacy compatibility,
 # and will use the initial_token as described below.
 #
-# Specifying initial_token will override this setting.
+# Specifying initial_token will override this setting on the node's initial start,
+# on subsequent starts, this setting will apply even if initial token is set.
 #
 # If you already have a cluster with 1 token per node, and wish to migrate to 
-# multiple tokens per node, see https://wiki.apache.org/cassandra/Operations
-#num_tokens: 256
+# multiple tokens per node, see http://wiki.apache.org/cassandra/Operations
+num_tokens: 256
 
 # initial_token allows you to specify tokens manually.  While you can use # it with
 # vnodes (num_tokens > 1, above) -- in which case you should provide a 
 # comma-separated list -- it's primarily used when adding nodes # to legacy clusters 
 # that do not have vnodes enabled.
 # initial_token:
 
-initial_token: ${cassandra.token}
-
-${cassandra.num.tokens}
-
+# See http://wiki.apache.org/cassandra/HintedHandoff
```

Comment:
Also broken link. This one may be better: https://wiki.apache.org/confluence/display/CASSANDRA2/HintedHandoff

Reply:
I just copied this file from the original distribution with some smaller changes from our side. I don't want to change issues which came from the original. This would make it hard to see difference, if we update cassandra.